index.html

```diff
@@ -9,13 +9,7 @@
   
   <link rel="stylesheet" href="mystyles.css">
 
-  <script type="text/javascript" src="/js/toMusic.js"></script>
-
-  <script type="text/javascript" src="/js/toShop.js"></script>
-
-  <script type="text/javascript" src="/js/toContact.js"></script>
-  
-  <script type="text/javascript" src="/js/toMailing.js"></script>
+  <script type="text/javascript" src="/js/index.js"></script>
```

Comment:
So far I don't think there is any need to have a javascript file. Typically though, 'index.js' will cover your basic needs for the site.